manager/integration/tests/test_ha.py

```diff
@@ -836,3 +836,107 @@ def test_rebuild_with_inc_restoration(
     delete_and_wait_pod(core_api, dr_pod_name)
     delete_and_wait_pvc(core_api, dr_pvc_name)
     delete_and_wait_pv(core_api, dr_pv_name)
+
+
+@pytest.mark.skip
+def test_restore_volume_with_invalid_backupstore(
+        client, core_api, volume_name, pod_make):  # NOQA
+    """
+    [HA] Test if the invalid backup target will lead to the restore volume
+    becoming Faulted, and if the auto salvage feature is disabled for
+    the faulted restore volume.
+
+    1. Enable auto-salvage.
+    2. Set a S3 backupstore. (Cannot use NFS server here before fixing #1295)
+    3. Create a volume then a backup.
+    4. Invalidate the target URL.
+       (e.g.: s3://backupbucket-invalid@us-east-1/backupstore-invalid)
+    5. Restore a volume from the backup.
+       (The fromBackup fields of the volume create API should consist of
+       the invalid target URL and the valid backup volume info)
+    6. Wait for the restore volume Faulted.
+    7. Check if the volume condition "restore":
+       `volume.conditions[restore].status == False &&
+       volume.conditions[restore].reason == "RestoreFailure".`
+    8. Check if the auto-salvage is disabled even if the setting is enabled.
+    9. Check if `volume.ready` is false
```

Comment:
Can we also add the following:

10. Verify if PV/PVC cannot be created from Longhorn.
11. Verify the faulted volume cannot be attached to a node.
12. Verify this faulted volume can be deleted

Reply:
YES, I will add it